tests/tests_integration/test_api/test_simulators/seed/data.py

```diff
@@ -25,6 +25,7 @@
     "simulator_model_external_id": f"py_sdk_integration_tests_model_{random_str}",
     "simulator_model_revision_external_id": f"py_sdk_integration_tests_model_{random_str}_v1",
     "simulator_model_file_external_id": "ShowerMixer_simulator_model_file_5",
+    "simulator_model_external_dependency_file_external_id": "Prosper_simulator_model_external_dependency_file",
```

Comment:
don't use Prosper references here. also use rename `.out` to something else (txt, csv, etc)